charts/vald/values-scylla.yaml

```diff
@@ -64,3 +64,17 @@ meta:
     config:
       hosts:
         - vald-scylla-cluster-dc0-rack0-0.scylla.svc.cluster.local
+
+initializer:
+  cassandra:
+    enabled: true
+    name: scylla-init
```

Comment:
It's not in the scope of this PR, I've found that it is not reflected to the job.yaml. Could you please fix it?
https://github.com/vdaas/vald/blob/a443ae89f1bdf47c7cd3752db303aac7fa35db05/charts/vald/templates/jobs/db/initialize/cassandra/job.yaml#L20
Also for mysql...
https://github.com/vdaas/vald/blob/a443ae89f1bdf47c7cd3752db303aac7fa35db05/charts/vald/templates/jobs/db/initialize/mysql/job.yaml#L20